versions/3.0.md

```diff
@@ -1152,6 +1152,51 @@ name: pet
 description: Pets operations
 ```
 
+#### <a name="examplesObject"></a>Examples Object
+
+Anywhere an `example` may be given, allow a $ref object.  This does mean that `example` can be either a string primitive or an object, like `additionalProperties`:
```

Comment:
@ePaul actually, in 2.0 it is the `default` field value has to be the same type as it's 'container' (be it a parameter or a model property). The examples have on such constraint.